src/js/components/Page/stories/CustomThemed/LeftColumn.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import {
+  Page,
+  Box,
+  Header,
+  Heading,
+  Paragraph,
+  Grid,
+  Card,
+  PageSection,
+  Grommet,
+} from 'grommet';
+
+const customTheme = {
+  page: {
+    testKind: {
+      align: 'start',
+      width: {
+        min: '200px',
+        max: '500px',
+      },
+      pad: {
+        small: 'medium',
+        medium: 'medium',
+        large: 'medium',
+      },
+    },
+  },
+};
+
+export const LeftColumn = () => (
+  <Grommet theme={customTheme}>
+    <Box background="#35bd55">
+      <Heading>Global Header</Heading>
+    </Box>
+    <Page kind="testKind">
+      <PageSection>
+        <Header>
+          <Heading>Heading</Heading>
+        </Header>
+      </PageSection>
+      <PageSection full background="pink">
```

Comment:
Theoretically, this should work for a background image as well. For peace of mind, can we do a background image as well?

Reply:
Yes, I added a background image to the narrow story in the latest commit